shared_libraries/orca_shared/recovery/shared_recovery.py

```diff
@@ -217,3 +217,50 @@ def post_entry_to_queue(
         raise Exception(
             f"Calculated MD5 of {md5_body} does not match SQS MD5 of {sqs_md5}"
         )
+
+
+def post_entry_to_standard_queue(
+    new_data: Dict[str, Any],
+    recovery_queue_url: str,
+) -> None:
+    """
+    Posts messages to the recovery standard SQS queue.
+    Args:
+        new_data: A dictionary representing the column/value pairs to write to the DB table.
+        recovery_queue_url: The SQS queue URL defined by AWS.
+    Raises:
+        None
+    """
+    body = json.dumps(new_data)
+
+    # TODO: pass AWS region value to function. SHOULD be gotten from environment
+    # higher up. Setting this to us-west-2 initially since that is where
+    # EOSDIS runs from normally. SEE ORCA-203 https://bugs.earthdata.nasa.ov/browse/ORCA-203
+    LOGGER.debug(
+        "Creating SQS resource for {recovery_queue_url}",
+        recovery_queue_url=recovery_queue_url,
+    )
+    mysqs_resource = boto3.resource("sqs", region_name=get_aws_region())
+    mysqs = mysqs_resource.Queue(recovery_queue_url)
+
+    md5_body = hashlib.md5(body.encode("utf8")).hexdigest()  # nosec
+
+    LOGGER.debug("Sending message to the QUEUE")
+    response = mysqs.send_message(
+        QueueUrl=recovery_queue_url,
+        MessageBody=body,
+    )
+    LOGGER.debug("SQS Message Response: {response}", response=json.dumps(response))
```

Comment:
Everything from here down could be put in a shared `check_sqs_response` method, but I'm also okay with keeping things fully separate, at least for now.

Reply:
sounds good.